tests/testthat/test-array.R

```diff
@@ -39,7 +39,7 @@ test_that("Basic types of length zero are passed correctly from R to .NET", {
 })
 
 # Same for length 1!
-# Investigate after https://github.com/Open-Systems-Pharmacology/rSharp/issues/35
+# https://github.com/Open-Systems-Pharmacology/rSharp/issues/108
```

Comment:
Can you also link this to the issue if it hasn't already been done. You can paste a link to a line of code.

Reply:
This might be a line from @PavelBal originally

Reply:
Yes this if from me, and it will not be fixed in this PR